vueui/src/main.js

```diff
@@ -95,15 +85,18 @@ if (document.getElementById("header")) {
 if (document.getElementById("dapp")) {
   new Vue({
     data: Store.state,
-    template: '<div id="debug"><h2>Debug this UI with inspector by opening URL in your browser:</h2><pre>{{url}}</pre><h2>Current data of UI:</h2><pre>{{ JSON.stringify(this.$root.$data, null, \'    \') }}</pre><button @click="stop()">STOP WTIME TRACKING</button></div>',
+    template: '<div id="debug" :class="shouldShow"><h2>Debug this UI with inspector by opening URL in your browser:</h2><pre>{{url}}</pre><h2>Current data of UI:</h2><pre>{{ JSON.stringify(this.$root.$data, null, \'    \') }}</pre><button @click="stop()">STOP WTIME TRACKING</button></div>',
     methods: {
       stop() {
         window.clearInterval(window.__wtimetimer)
       }
     },
     computed: {
       url() {
-        return window.location.href
+        return window.location.href + '?ext'
+      },
+      shouldShow() {
+        return this.$root.$data.debug_view ? "uiDebug" : ""
```

Comment:
This may have been why it wasn't reactive
```suggestion
        return this.debug_view ? "uiDebug" : ""
```

Reply:
unfortunately doesn't seem to have changed anything